README.md

```diff
@@ -18,7 +18,7 @@ Please see our [documentation](https://opendistro.github.io/for-elasticsearch-do
 ## Setup
 
 1. Download Elasticsearch for the version that matches the [Kibana version specified in package.json](./package.json#L9).
-1. Download and install the appropriate [Open Distro for Elasticsearch Alerting plugin](https://github.com/opendistro-for-elasticsearch/alerting).
+1. Download and install the appropriate [Open Distro  Alerting plugin](https://github.com/opendistro-for-elasticsearch/alerting).
```

Comment:
Super nitpick but there's an extra space between "Distro" and "Alerting"

Reply:
Fixed